lua/sttusline/components/file-type.lua

```diff
@@ -0,0 +1,21 @@
+return {
+	name = "file-type",
+	event = {
+		"BufEnter",
+		"BufWritePost",
+		"BufReadPost",
+	},
+	configs = {
+		style = "default",
+	},
+	padding = 2,
+	user_event = "VeryLazy",
+	update = function()
+		local icon = require("nvim-web-devicons").get_icon(
```

Comment:
and alternative in the case that `nvim-web-devicons` is not available is to use:

```lua
update = function()
    local ft = vim.bo.filetype
    local has_devicons, web_devicons = pcall(require, "nvim-web-devicons")
    local icon

    if has_devicons then
        icon = web_devicons.get_icon(
            vim.fn.expand "%:t",
            vim.fn.expand "%:e",
            { default = true }
        )
    else
        icon = ft
    end

    return string.format("%s", icon)
end,
```

Reply:
Yah, feel free to add it if needed.

Reply:
Done in 7cfaecd

Reply:
Is there anything else needed to merge this pr @sontungexpt ?